content/news/release-20170527.md

```diff
@@ -0,0 +1,80 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+We are today pushing a new bugfix release that addresses a collection of issues that were missed during the playtesting phase of release-20170421.
+
+The main changes in this hotfix are:
+
+* Disabled HiDPI support on Windows, fixing the graphical glitches this caused on non-HiDPI displays.
+* Fixes cursor glitches introduced with the new Mono 5.0 runtime.
+* Fixes graphical glitches with the Fullscreen (Legacy) mode on macOS.
+
+A collection of smaller fixes are listed in the  [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/).
+
+Head on over to our [download page](/download/) to grab the new release, and spread the word!
+
+</div>
+</div>
+The  [original release-announcement](/news/release-20170421/) for release-20170421 is included below:
+
+<hr>
```

Comment:
Everything below this line is a direct copy from the original post, so no need to proof read.